Sources/ObjCSupport/LOKLabelLayout.swift

```diff
@@ -9,6 +9,9 @@
 import UIKit
 
 @objc open class LOKLabelLayout: LOKBaseLayout {
+    @objc public let attributedString: NSAttributedString?
+    @objc public let text: String?
```

Comment:
Should be consistent in the naming of these two properties. Either both end in `text` or both end in `string`.

Reply:
Renamed variables.